saleor/checkout/utils.py

```diff
@@ -1025,3 +1025,22 @@ def get_checkout_metadata(checkout: "Checkout"):
         return checkout.metadata_storage
     else:
         return CheckoutMetadata(checkout=checkout)
+
+
+def log_address_if_validation_skipped_for_checkout(
+    checkout_info: "CheckoutInfo", logger
+):
+    address = get_address_for_checkout_taxes(checkout_info)
+    if address and address.validation_skipped:
```

Comment:
Does it mean, that we will always log the address ID, when the validation was skipped? It means multiple warning log records for a single checkout. 
I think, that we should log the warning only when: validation is skipped and the address looks incorrect from our library perspective.

Reply:
The name `validation_skipped` is misleading at the moment and I am working on renaming it in separate PR.  `validation_skipped` will be set to True only when address contains some field with invalid format.